apps/database/postgres/ormconfig.ts

```diff
@@ -0,0 +1,18 @@
+import { ConnectionOptions } from "typeorm"
+
+const config: ConnectionOptions = {
```

Comment:
you added this but you're not using it in the app? also where is synchronize?

Reply:
So this file is used only when running the migration which it's path is specified in the npm script command. Locally migrations will be done automatically as they will be synchronized from the model. I'm not sure if that's the right approach though. Maybe we should have the same process locally as in production? We could run the migrations on API startup to ensure we're always up to date.

What do you think?

Reply:
We could. But my main concern is that we are duplicating the config here and in the api. We should use the same for both to avoid modification errors

Reply:
That is a very valid concern.
I had tried importing the configuration file from the API, but since I need to target commonJS, it doesn't support the paths provided in the `tsconfig.base.json`. One way this could work would be to flip things around where `ormconfig.ts` would be the source of truth, and the API configuration file would import the `ormconfig.ts` file properties.

Makes sense?

Reply:
Yep, I agree with the swap